.github/workflows/ci.yml

```diff
@@ -96,7 +96,7 @@ jobs:
         python-version: ${{ matrix.python.action }}
 
     - name: Install dependencies
-      run: python -m pip install --upgrade pip tox codecov
+      run: python -m pip install --upgrade pip tox codecov coverage[toml]
```

Comment:
I wish pip had something like `pip install --only-deps` 

But maybe, we can just do `pip install .[dev]` and run the tests without tox.